delivery_package_fee/tests/test_package_fee.py

```diff
@@ -81,7 +81,7 @@ def _create_sale(cls):
     def _add_sale_carrier(cls, sale, carrier):
         delivery_wizard = Form(
             cls.env["choose.delivery.carrier"].with_context(
-                {"default_order_id": sale.id, "default_carrier_id": carrier.id}
+                **{"default_order_id": sale.id, "default_carrier_id": carrier.id}
```

Comment:
You can avoid such notation setting parameters directly:

`(...).with_context(default_order_id=sale.id, ...)`